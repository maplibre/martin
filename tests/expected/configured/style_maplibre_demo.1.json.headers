content-encoding: br
content-type: application/json
etag: W/"1050-ndsWDIE-ORCeUzSbybafug=="
transfer-encoding: chunked
vary: accept-encoding, Origin, Access-Control-Request-Method, Access-Control-Request-Headers
