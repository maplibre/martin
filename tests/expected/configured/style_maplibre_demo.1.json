{
  "bearing": 0,
  "center": [
    17.65431710431244,
    32.954120326746775
  ],
  "glyphs": "https://demotiles.maplibre.org/font/{fontstack}/{range}.pbf",
  "id": "43f36e14-e3f5-43c1-84c0-50a9c80dc5c7",
  "layers": [
    {
      "filter": [
        "all"
      ],
      "id": "background",
      "layout": {
        "visibility": "visible"
      },
      "maxzoom": 24,
      "paint": {
        "background-color": "#D8F2FF"
      },
      "type": "background"
    },
    {
      "filter": [
        "all"
      ],
      "id": "coastline",
      "layout": {
        "line-cap": "round",
        "line-join": "round",
        "visibility": "visible"
      },
      "maxzoom": 24,
      "minzoom": 0,
      "paint": {
        "line-blur": 0.5,
        "line-color": "#198EC8",
        "line-width": {
          "stops": [
            [
              0,
              2
            ],
            [
              6,
              6
            ],
            [
              14,
              9
            ],
            [
              22,
              18
            ]
          ]
        }
      },
      "source": "maplibre",
      "source-layer": "countries",
      "type": "line"
    },
    {
      "filter": [
        "all"
      ],
      "id": "countries-fill",
      "layout": {
        "visibility": "visible"
      },
      "maxzoom": 24,
      "paint": {
        "fill-color": [
          "match",
          [
            "get",
            "ADM0_A3"
          ],
          [
            "ARM",
            "ATG",
            "AUS",
            "BTN",
            "CAN",
            "COG",
            "CZE",
            "GHA",
            "GIN",
            "HTI",
            "ISL",
            "JOR",
            "KHM",
            "KOR",
            "LVA",
            "MLT",
            "MNE",
            "MOZ",
            "PER",
            "SAH",
            "SGP",
            "SLV",
            "SOM",
            "TJK",
            "TUV",
            "UKR",
            "WSM"
          ],
          "#D6C7FF",
          [
            "AZE",
            "BGD",
            "CHL",
            "CMR",
            "CSI",
            "DEU",
            "DJI",
            "GUY",
            "HUN",
            "IOA",
            "JAM",
            "LBN",
            "LBY",
            "LSO",
            "MDG",
            "MKD",
            "MNG",
            "MRT",
            "NIU",
            "NZL",
            "PCN",
            "PYF",
            "SAU",
            "SHN",
            "STP",
            "TTO",
            "UGA",
            "UZB",
            "ZMB"
          ],
          "#EBCA8A",
          [
            "AGO",
            "ASM",
            "ATF",
            "BDI",
            "BFA",
            "BGR",
            "BLZ",
            "BRA",
            "CHN",
            "CRI",
            "ESP",
            "HKG",
            "HRV",
            "IDN",
            "IRN",
            "ISR",
            "KNA",
            "LBR",
            "LCA",
            "MAC",
            "MUS",
            "NOR",
            "PLW",
            "POL",
            "PRI",
            "SDN",
            "TUN",
            "UMI",
            "USA",
            "USG",
            "VIR",
            "VUT"
          ],
          "#C1E599",
          [
            "ARE",
            "ARG",
            "BHS",
            "CIV",
            "CLP",
            "DMA",
            "ETH",
            "GAB",
            "GRD",
            "HMD",
            "IND",
            "IOT",
            "IRL",
            "IRQ",
            "ITA",
            "KOS",
            "LUX",
            "MEX",
            "NAM",
            "NER",
            "PHL",
            "PRT",
            "RUS",
            "SEN",
            "SUR",
            "TZA",
            "VAT"
          ],
          "#E7E58F",
          [
            "AUT",
            "BEL",
            "BHR",
            "BMU",
            "BRB",
            "CYN",
            "DZA",
            "EST",
            "FLK",
            "GMB",
            "GUM",
            "HND",
            "JEY",
            "KGZ",
            "LIE",
            "MAF",
            "MDA",
            "NGA",
            "NRU",
            "SLB",
            "SOL",
            "SRB",
            "SWZ",
            "THA",
            "TUR",
            "VEN",
            "VGB"
          ],
          "#98DDA1",
          [
            "AIA",
            "BIH",
            "BLM",
            "BRN",
            "CAF",
            "CHE",
            "COM",
            "CPV",
            "CUB",
            "ECU",
            "ESB",
            "FSM",
            "GAZ",
            "GBR",
            "GEO",
            "KEN",
            "LTU",
            "MAR",
            "MCO",
            "MDV",
            "NFK",
            "NPL",
            "PNG",
            "PRY",
            "QAT",
            "SLE",
            "SPM",
            "SYC",
            "TCA",
            "TKM",
            "TLS",
            "VNM",
            "WEB",
            "WSB",
            "YEM",
            "ZWE"
          ],
          "#83D5F4",
          [
            "ABW",
            "ALB",
            "AND",
            "ATC",
            "BOL",
            "COD",
            "CUW",
            "CYM",
            "CYP",
            "EGY",
            "FJI",
            "GGY",
            "IMN",
            "KAB",
            "KAZ",
            "KWT",
            "LAO",
            "MLI",
            "MNP",
            "MSR",
            "MYS",
            "NIC",
            "NLD",
            "PAK",
            "PAN",
            "PRK",
            "ROU",
            "SGS",
            "SVN",
            "SWE",
            "TGO",
            "TWN",
            "VCT",
            "ZAF"
          ],
          "#B1BBF9",
          [
            "ATA",
            "GRL"
          ],
          "#FFFFFF",
          "#EAB38F"
        ]
      },
      "source": "maplibre",
      "source-layer": "countries",
      "type": "fill"
    },
    {
      "id": "countries-boundary",
      "layout": {
        "line-cap": "round",
        "line-join": "round",
        "visibility": "visible"
      },
      "maxzoom": 24,
      "paint": {
        "line-color": "rgba(255, 255, 255, 1)",
        "line-opacity": {
          "stops": [
            [
              3,
              0.5
            ],
            [
              6,
              1
            ]
          ]
        },
        "line-width": {
          "stops": [
            [
              1,
              1
            ],
            [
              6,
              2
            ],
            [
              14,
              6
            ],
            [
              22,
              12
            ]
          ]
        }
      },
      "source": "maplibre",
      "source-layer": "countries",
      "type": "line"
    },
    {
      "filter": [
        "all",
        [
          "!=",
          "name",
          "International Date Line"
        ]
      ],
      "id": "geolines",
      "layout": {
        "visibility": "visible"
      },
      "maxzoom": 24,
      "paint": {
        "line-color": "#1077B0",
        "line-dasharray": [
          3,
          3
        ],
        "line-opacity": 1
      },
      "source": "maplibre",
      "source-layer": "geolines",
      "type": "line"
    },
    {
      "filter": [
        "all",
        [
          "!=",
          "name",
          "International Date Line"
        ]
      ],
      "id": "geolines-label",
      "layout": {
        "symbol-placement": "line",
        "text-field": "{name}",
        "text-font": [
          "Open Sans Semibold"
        ],
        "text-size": {
          "stops": [
            [
              2,
              12
            ],
            [
              6,
              16
            ]
          ]
        },
        "visibility": "visible"
      },
      "maxzoom": 24,
      "minzoom": 1,
      "paint": {
        "text-color": "#1077B0",
        "text-halo-blur": 1,
        "text-halo-color": "rgba(255, 255, 255, 1)",
        "text-halo-width": 1
      },
      "source": "maplibre",
      "source-layer": "geolines",
      "type": "symbol"
    },
    {
      "filter": [
        "all"
      ],
      "id": "countries-label",
      "layout": {
        "text-field": {
          "stops": [
            [
              2,
              "{ABBREV}"
            ],
            [
              4,
              "{NAME}"
            ]
          ]
        },
        "text-font": [
          "Open Sans Semibold"
        ],
        "text-max-width": 10,
        "text-size": {
          "stops": [
            [
              2,
              10
            ],
            [
              4,
              12
            ],
            [
              6,
              16
            ]
          ]
        },
        "text-transform": {
          "stops": [
            [
              0,
              "uppercase"
            ],
            [
              2,
              "none"
            ]
          ]
        },
        "visibility": "visible"
      },
      "maxzoom": 24,
      "minzoom": 2,
      "paint": {
        "text-color": "rgba(8, 37, 77, 1)",
        "text-halo-blur": {
          "stops": [
            [
              2,
              0.2
            ],
            [
              6,
              0
            ]
          ]
        },
        "text-halo-color": "rgba(255, 255, 255, 1)",
        "text-halo-width": {
          "stops": [
            [
              2,
              1
            ],
            [
              6,
              1.6
            ]
          ]
        }
      },
      "source": "maplibre",
      "source-layer": "centroids",
      "type": "symbol"
    }
  ],
  "metadata": {
    "maptiler:copyright": "This style was generated on MapTiler Cloud. Usage is governed by the license terms in https://github.com/maplibre/demotiles/blob/gh-pages/LICENSE",
    "openmaptiles:version": "3.x"
  },
  "name": "MapLibre",
  "pitch": 0,
  "projection": {
    "type": "globe"
  },
  "sources": {
    "maplibre": {
      "type": "vector",
      "url": "https://demotiles.maplibre.org/tiles/tiles.json"
    }
  },
  "version": 8,
  "zoom": 0.8619833357855968
}
